images/Dockerfile

```diff
@@ -16,350 +16,499 @@
 #
 # ***************************************************************
 
-ARG BASE_YUM_REPO=release
-ARG BASE_OSG_SERIES=23
+# For selecting OSG-maintained packages.
 ARG BASE_OS=el9
+ARG OSG_SERIES=23
+ARG OSG_REPO=release
+
+# For compiling Pelican.
+ARG GO_VER=1.21.6
+ARG NODEJS_VER=20
+
+# For compiling LotMan.
+ARG JSON_VERSION="v3.11.3"
```

Comment:
Looks like the OSG koji has basically this version (`v3.11.2`, at least) already built and available.  Let's use that instead.